viewing_party/party.py

```diff
@@ -0,0 +1,179 @@
+def create_movie(title, genre, rating):        
+    if title and genre and rating:
+        movie_dict ={}
+        movie_dict["title"] = title
+        movie_dict["genre"] = genre
+        movie_dict["rating"] = rating
+        return movie_dict
+    elif not title or genre or rating:
+        return None
+
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+    return(user_data)
+
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+
+    return(user_data)
+
+def watch_movie(user_data, title):
+    for movie in user_data["watchlist"]:
+        if movie["title"] == title:
+            user_data["watchlist"].remove(movie)
+            user_data["watched"].append(movie)
+
+    return user_data
```

Comment:
Be careful about modifying a collection as you're iterating through it (watchlist). For this example, we're only removing a single instance of a movie, however, if there are multiple instances of that movie that could cause the rest of the items in the list to shift position and potentially skip over other items as the loop continues.

This case is fine, but let's proceed with caution when modifying a collection while iterating through it.

Here's more info: https://thispointer.com/python-remove-elements-from-a-list-while-iterating/